### PR TITLE
Reader implementation for webvtt

### PR DESCRIPTION
**Pull request by ankita-vimeo**
for review please refer https://github.com/pbs/pycaption/blob/main/pycaption/webvtt.py#L44-L169.
Let me know if this a really big PR, I can split it in multiple PRs.

### CODE REVIEW

File: caption.go [hunk 1]
```diff
@@ -47,19 +47,19 @@ func (c Caption) FormatEnd() string {
 	return formatTimestamp(c.End, ".")
 }
 
-func formatTimestamp(timestamp float64, sep string) string {
-	value := int(timestamp / 1000)
+func formatTimestamp(timestamp *float64, sep string) string {
```
Comment (AdamNastasia) [inline caption.go:50]: is there any way we can leverage the go time package in this function?

Comment (ankita-vimeo) [reply caption.go:50]: Do you mean instead of using pointer here?

Comment (AdamNastasia) [reply caption.go:50]: ah, I mean to parse/format the float64 timestamp, it looks to be done manually in the function

Comment (ankita-vimeo) [reply caption.go:50]: I can create a separate PR for just this.

**Review by thiagopnts — APPROVED** [review]

**Pull request merged**